scenarioo-client/app/step/step.controller.ts

```diff
@@ -141,9 +141,9 @@ function StepController($scope, $routeParams, $location, $route, StepResource, S
                     $scope.stepNotFound = true;
                     $scope.httpResponse = {
                         status: error.status,
-                        method: error.config.method,
-                        url: error.config.url,
-                        data: error.data,
+                        method: 'GET',
```

Comment:
Why can we now fix this?

Reply:
This information was not part of the error object. On the other hand, I noticed on line 127 that "StepResource.get" is called, so I'm quite certain that this is always a GET request.